plugins/woocommerce-blocks/assets/js/data/shared-controls.ts

```diff
@@ -126,7 +126,10 @@ export const apiFetchWithHeadersControl = ( options: APIFetchOptions ) =>
 	} as const );
 
 // List of paths which should not be batched.
-const preventBatching = [ '/wc/store/v1/cart/select-shipping-rate' ];
+const preventBatching = [
+	'/wc/store/v1/cart/select-shipping-rate',
+	'/wc/store/v1/checkout?calc_totals=true',
```

Comment:
Prevent batching on the PUT checkout endpoint so that we can cancel it. Batch uses an external library which from what I can tell doesn't support cancelling requests.

Reply:
Should we just avoid batching on all `/wc/store/v1/checkout` endpoints?

Reply:
Yep good idea, done in https://github.com/woocommerce/woocommerce/pull/53553/commits/c5da2ce2286155a5714284ef25b99aa6f12ed79d